sim/battle.ts

```diff
@@ -1552,6 +1552,43 @@ export class Battle {
 			this.add('bigerror', `You will auto-tie if the battle doesn't end in ${turnsLeftText} (on turn 1000).`);
 		}
 
+		// Gen 1 Endless Battle Clause triggers
+		if (this.gen <= 1) {
+			let onlyTransform = true;
+			let onlyGhostsWithStruggle = true;
+			let onlyFrozen = true;
```

Comment:
I think this should be one variable, rather than three. Having just one `nothingCanMove` variable would catch cases such as "a frozen ghost vs a pokemon with only Struggle".

Reply:
yeah sounds good.

Reply:
I think I did it, albeit in a different direction than you anticipated.